napari/layers/image/_image_slice.py

```diff
@@ -25,18 +25,21 @@ class ImageSlice:
     thumbnail : ImageView
         The smaller thumbnail image for this slice.
 
-    Example
-    -------
-        # Create with some default image.
-        image_slice = ImageSlice(default_image)
-
-        # Set raw image or thumbnail, viewable is computed.
-        image_slice.image = raw_image
-        image_slice.thumbnail = raw_thumbnail
-
-        # Access the viewable images.
-        draw_image(image_slice.image.view)
-        draw_thumbnail(image_slice.thumbnail.view)
+    Examples
+    --------
+    Create with some default image:
+
+    >> image_slice = ImageSlice(default_image)
+
+    Set raw image or thumbnail, viewable is computed.:
+
+    >> image_slice.image = raw_image
+    >> image_slice.thumbnail = raw_thumbnail
```

Comment:
fwiw these should be triple `>>>` as the standard Python prompt.

Additionally, ideally all of our doctests will be executable and thus tested, which helps prevent them falling out of sync. When run, the are executed in the namespace of the module in which they live. So you'd want to add `  # doctest: +SKIP` at the end of lines that are meant to be explanatory rather than executed, which I think applies to the lines below?

Reply:
I think I'll remove these `Examples` sections from the new PR for a few reasons:

1. I invented the idea of an **Example** section without knowing there was a standard that included doctest.

2. There's nothing to show here, there are no return values. It's pretty pointless.

3. No one is going to use these classes except where we do in the real system. No reason to pretend people will. Better to just explain how they work in regular docs.

I think an `Examples` section makes sense for some classes but not these.